pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/template/render/LatexVisualITest.kt

```diff
@@ -2,125 +2,319 @@ package no.nav.pensjon.brev.template.render
+            brevtype = brevtype,
+            attachments = attachments,
+            outlineInit = outlineInit,
+            title = title ?: testName
+        )
+    }
 
-    private fun render(overrideName: String? = null, outlineInit: OutlineOnlyScope<LangBokmal, Unit>.() -> Unit) {
-        val testName = overrideName ?: StackWalker.getInstance().walk { frames -> frames.skip(2).findFirst().map { it.methodName }.orElse("") }
-
-        val template = createTemplate(testName, Unit::class, languages(Bokmal), LetterMetadata(
-            testName,
-            false,
-            LetterMetadata.Distribusjonstype.VEDTAK,
-            LetterMetadata.Brevtype.VEDTAKSBREV
-        )) {
-            title {
-                text(Bokmal to testName)
-            }
-            outline { outlineInit() }
-        }
-        val letter = Letter(template, Unit, Bokmal, Fixtures.fellesAuto)
-        letter.renderTestPDF(testName, Path.of("build/test_visual/pdf"))
+    private fun renderTestVedlegg(
+        includeSakspart: Boolean,
+        testName: String? = null,
+        felles: Felles? = null,
+        vedleggOutlineInit: OutlineOnlyScope<LangBokmal, EmptyBrevdata>.() -> Unit,
+    ) {
+        renderTestVedleggPdf(
+            outputFolder = "test_visual/pdf",
+            testName = testName ?: StackWalker.getInstance()
+                .walk { frames -> frames.skip(2).findFirst().map { it.methodName }.orElse("") },
+            includeSakspart = includeSakspart,
+            outlineInit = vedleggOutlineInit,
+            felles = felles,
+        )
     }
 
-    private fun ParagraphOnlyScope<LangBokmal, Unit>.ipsumText() = text(Bokmal to lipsums.first())
+
+    private fun ParagraphOnlyScope<LangBokmal, EmptyBrevdata>.ipsumText() = text(Bokmal to lipsums[1])
 
     @Test
-    fun `Paragraph after letter title`() {
+    fun `two paragraphs in a row`() {
         render {
             paragraph { ipsumText() }
+            paragraph { ipsumText() }
         }
     }
 
     @Test
-    fun `Title1 after letter title`() {
+    fun fonts() {
         render {
-            title1 { text(Bokmal to "A title1 text") }
+            paragraph {
+                text(Bokmal to "The quick brown fox jumps over the lazy dog. ", FontType.PLAIN)
+                text(Bokmal to "The quick brown fox jumps over the lazy dog. ", FontType.ITALIC)
+                text(Bokmal to "The quick brown fox jumps over the lazy dog. ", FontType.BOLD)
+            }
         }
     }
 
     @Test
-    fun `Title2 after letter title`() {
-        render {
-            title2 { text(Bokmal to "A title2 text") }
+    fun `verge foersteside`() {
+        render(
+            felles = Fixtures.felles.copy(
+                vergeNavn = "Verge vergeson"
+            )
+        ) {
+            testTitle1()
+            paragraph { ipsumText() }
         }
     }
 
     @Test
-    fun `Title1 after letter title followed by paragraph`() {
+    fun `form elementer`() {
```

Comment:
Burde kanskje splittes i flere tester?
Slik det er nå så vil det bli vanskelig å se i diffen om man kun har endret på det man forsøkte å endre på. F.eks om man skal endre på vspace mellom to `choice` så vil det påvirke hele resten av pdfen.

Reply:
Done